docs/framework/winforms/disable-dpi-awareness-visual-studio.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Disable DPI-awareness in Visual Studio
+ms.date: 08/14/2018
+ms.prod: visual-studio-dev15
+ms.technology: vs-ide-designers
+ms.topic: conceptual
+author: gewarren
+ms.author: gewarren
+manager: douge
+---
+# Disable DPI-awareness in Visual Studio
+
+Visual Studio is a dots per inch (DPI) aware application, which means displays scale automatically. If an application states that it's not DPI-aware, the operating system scales the application as a bitmap. This behavior is also called DPI virtualization. The application still thinks that it's running at 100% scaling, or 96 dpi.
+
+## Rendering issue
+
+The **Windows Forms Designer** in Visual Studio doesn't currently have scaling support. This can cause display issues when you open forms in the **Windows Forms Designer** on HDPI monitors.
+
+For example, the first image that follows shows **Windows Forms Designer** on a monitor at 100% scaling. The second image shows what the form looks like when it's opened on a high dots per inch (HDPI) monitor:
+
+![Windows Forms Designer scaling at 100%](media/scaling-100-percent.png)
+
+![Windows Forms Designer on HDPI monitor](media/win-forms-designer-hdpi.png)
```

Comment:
I think it will look better if the pictures are the same size

Reply:
Looking at the images it does seem as if they're not super high quality. Should we try getting new images that are higher quality?  I don't think we _can_ make them the same size because the scaling is what changes the sizes.